features/search/components/ProfileSearchItem.tsx

```diff
@@ -37,37 +39,39 @@ export function ProfileSearchItem({
   ];
 
   return (
-    <View key={address} style={themed($container)}>
-      <View style={themed($left)}>
-        <Avatar
-          uri={preferredAvatar}
-          size={theme.spacing["3xl"]}
-          style={themed($avatar)}
-          name={preferredName}
-        />
-        <View style={themed($textContainer)}>
-          <Text preset="bodyBold" numberOfLines={1}>
-            {preferredName}
-          </Text>
-          {primaryNamesDisplay.length > 0 && (
-            <Text preset="formLabel" numberOfLines={1}>
-              {primaryNamesDisplay.join(" | ")}
+    <Pressable
+      onPress={() => {
+        logger.info("ProfileSearchResultListItem onPress", {
+          address,
+          socials,
+        });
+        handleSearchResultItemPress({
+          address,
+          /** todo probably just address */ socials,
+        });
+      }}
+    >
+      <View key={address} style={themed($container)}>
```

Comment:
Not sure why we have `key={address}`

Reply:
these will be rendered in a list so I'd imagine that's why that was there (again - copied)